src/plumtree_broadcast.erl

```diff
@@ -645,7 +657,7 @@ send(Msg, Mod, P) ->
                                       partisan_peer_service),
     PeerServiceManager = PeerService:manager(),
     instrument_transmission(Msg, Mod),
-    PeerServiceManager:forward_message(P, ?SERVER, Msg).
+    ok = PeerServiceManager:forward_message(P, ?SERVER, Msg).
```

Comment:
But, won't this crash the caller?  I'm wondering if this should return error instead?